slave-new/process-voms/bin/process-voms.sh

```diff
@@ -1,131 +1,12 @@
 #!/bin/bash
 
 # Script for managing user membership in VOs
-# Return values:
-# 0 ... OK
-# 1 ... Single user issue
-# 2 ... VO-wide issue
-# 3 ... VO-availability issue (VO does not exist or is inaccessible)
 
-PROTOCOL_VERSION='3.0.0'
-
-function log_both {
-	logger Perun:$@
-	echo $@ 1>&2
-}
+PROTOCOL_VERSION='3.1.0'
 
 function process {
-	### Status codes
-	I_CHANGED=(0 "${DST_FILE} updated")
-	I_NOT_CHANGED=(0 "${DST_FILE} has not changed")
-
-	### Error codes ###
-	E_VOMS_DELETE=(50 'Error during VOMS delete-user operation')
-	E_VOMS_CREATE=(50 'Error during VOMS create-user operation')
-
-	FROM_PERUN="${WORK_DIR}/voms"
-
-	RETVAL=0
-
-	create_lock
```

Comment:
Code with creating lock should stay there. It is protection before Race condition between two and more slave scripts.